src/lib/extract/extractFilter.ts

```diff
@@ -53,3 +55,27 @@ export const extractFeColorMatrix = (
 
   return extracted;
 };
+
+export const extractFeGaussianBlur = (
+  props: FeGaussianBlurComponentProps
+): FeGaussianBlurNativeProps => {
+  const extracted: FeGaussianBlurNativeProps = {};
+
+  if (props.in) extracted.in1 = props.in;
+  if (props.stdDeviation !== undefined) {
```

Comment:
What if it is undefined?

Reply:
Hmm actually this check may not be needed as `defaultProps` should be applied in that case (with `stdDeviation: 0`)

Reply:
Removed in https://github.com/software-mansion/react-native-svg/pull/2352/commits/8ffa742efeb7dac6511943813a2b38e4a2fae0b7